app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+
+angular.module("openshiftConsole")
+  .directive("oscPersistentVolumeClaim", function($filter){
+    return {
+      restrict: 'E',
+      scope: {
+        claim: "=model",
+        showNameInput: "=",
+        showAmountInput:"="
+      },
+      templateUrl: 'views/directives/osc-persistent-volume-claim.html',
+      link: function(scope) {
+              var amountAndUnit = $filter('amountAndUnit');
+              var humanizeUnit = $filter('humanizeUnit');
+
+              // Create a unique ID for `label for` and `aria-describedby` attributes.
+              scope.id = _.uniqueId('storage-size-');
```

Comment:
This was part of the compute-resource directive since it was included in the same page multiple times. I'd just remove scope.id here and give each element its own ID. The create PVC will be used once on any page. If you do use scope.id, you should use it for all fields, not just one or two. Easier to remove it though.